canvasapi/course.py

```diff
@@ -2347,6 +2347,80 @@ def list_blueprint_subscriptions(self, **kwargs):
             kwargs=combine_kwargs(**kwargs)
         )
 
+    def list_content_exports(self, **kwargs):
+        """
+        Return a paginated list of the past and pending content export jobs for a course.
+
+        :calls: `GET /api/v1/courses/:course_id/content_exports\
+        <https://canvas.instructure.com/doc/api/content_exports.html#method.content_exports_api.index>`_
+
+        :rtype: :class:`canvasapi.paginated_list.PaginatedList` of
+            :class:`canvasapi.content_export.ContentExport`
+        """
+        from canvasapi.content_export import ContentExport
+
+        return PaginatedList(
+            ContentExport,
+            self._requester,
+            'GET',
+            'courses/{}/content_exports'.format(
+                self.id
+            ),
+            kwargs=combine_kwargs(**kwargs)
+        )
+
+    def show_content_export(self, content_export, **kwargs):
```

Comment:
To be consistent with other method names, please change this to `get_content_export`.  Do the same on the `User` and `Group` classes.